### ISSUE TITLE
Theme broken on Firefox 108

### ISSUE THREAD

**Issue by Gilibenzio** [witalihirsch/Mono-firefox-theme#5]
The theme has **completely stopped working** on Firefox 108, it's completely broken.
At this time release of Firefox, including the stable one is at 108 so the theme is not usable anymore. 
![image](https://user-images.githubusercontent.com/73967962/207380604-0cf5d95e-6bf8-49b6-bb42-cad88e33b169.png)

I attach a link to the same bug on [firefox-gnome-theme](https://github.com/rafaelmardojai/firefox-gnome-theme/issues/504) because it's broken too and it can provide helpful informations:

https://github.com/rafaelmardojai/firefox-gnome-theme/issues/504

Thank you!

**Comment by witalihirsch** [comment]
thanks for reporting the issue. i'll try to solve it

**Comment by Ziiga** [comment]
I don't know how much this helps but by messing around with the .css files i found out that if you remove `@import "parts/popups.css";` from gnome-theme.css the issue seems to disappear. 

Will post if i figure out more.

**Comment by witalihirsch** [comment]
> I don't know how much this helps but by messing around with the .css files i found out that if you remove `@import "parts/popups.css";` from gnome-theme.css the issue seems to disappear.
> 
> Will post if i figure out more.

yes, the problem was in the popup.css file. everything is fixed and works for the new version as well as for the old ones

**Comment by witalihirsch** [comment]
thanks for the feedback

**Comment by Gilibenzio** [comment]
> thanks for the feedback

Thank you very much for solving the problem so quickly!
It's nice to see a developer so quick and helpful
